cypress/integration/updates/simulator-tooltip.spec.js

```diff
@@ -0,0 +1,76 @@
+/** This test needs to run AFTER create_slice. const values below are copied from there. */
+describe("simulator tooltip", () => {
```

Comment:
nice, I'm not sure how to ensure it runs after create slice aside from adding it to the bottom of create slice or as part of craete slice.

Although it might be easier to write a component test like what we have in slice-machine/test/...

Reply:
yeaaah, right now it _is_ run after the create slice but it's definitely something that needs to change. We should move forward with a new e2e test process very soon